### ISSUE TITLE
Accessibility issues: Scatter plot chart & Bar chart are not accessible through keyboard.

### ISSUE THREAD

**Issue by vijaykumar-peddamuthu** [plotly/plotly.js#4264]
We are using Scatter plot chart & Bar chart in our application. When we tested through keyboard none of the controls accessible. Please find below details of them and provide possible fix.

**1. Scatter plot chart**
        
![image](https://user-images.githubusercontent.com/46993098/66474600-46d43700-eaaf-11e9-97bf-7a99c50147d9.png)

Markup:
<Plot
	onRelayout={data => this.handleRelayout(data)}
	useResizeHandler
	type={'scatter'}
	layout={this.layout}
	style={{ width: '99%', height: '425px' }}
	data={this.prepData(chartData)}
	onClick={data => this.handlePlotClick(data)}
	onHover={() => {
	  // eslint-disable-next-line
	  this.dragLayer = document.getElementsByClassName('nsewdrag')[0];
	  this.dragLayer.style.cursor = 'pointer';
	}}
	onUnhover={() => {
	  // eslint-disable-next-line
	  this.dragLayer = document.getElementsByClassName('nsewdrag')[0];
	  this.dragLayer.style.cursor = '';
}} />

**2. Bar Chart** 
        
![image](https://user-images.githubusercontent.com/46993098/66474649-55bae980-eaaf-11e9-8d00-3a3e9d4c77be.png)

Markup:
<Plot
	onRelayout={data => this.handleRelayout(data)}
	useResizeHandler
	type={'bar'}
	layout={this.layout}
	style={{ width: '99%', height: '425px' }}
	data={this.prepData(chartData)}
	onHover={() => {
	  const dragLayer = document.getElementsByClassName('nsewdrag')[0];
	  dragLayer.style.cursor = 'pointer';
	}}
	onUnhover={() => {
	  const dragLayer = document.getElementsByClassName('nsewdrag')[0];
	  dragLayer.style.cursor = '';
	}}
	onClick={(data) => {
	  const colors = [];
	  const clickedPoint = data.points[0].pointNumber;
	  for (let i = 0; i < data.points[0].data.y.length; i += 1) {
		if (i === clickedPoint) {
		  colors[i] = processStatusColor.SELECTED;
		} else {
		  colors[i] = this.state.defaultColor[i];
		}
	  }
	  this.chartRelayout(data);
	  this.setState({
		currentColorsArray: colors,
		barClicked: data.points[0].pointNumber,
		showActivitySummary: true
	  });
	}} />

**Comment by etpinard** [comment]
Duplicate of https://github.com/plotly/plotly.js/issues/562

**Comment by anupsingh-anup** [comment]
#562 is still in open state and is any idea when it will be closed ? i see its still in discussion and only zoom-in/zoom-out feature implemented.please provide timeline @etpinard
